packages/common/src/types.ts

```diff
@@ -288,6 +297,11 @@ export interface IHubContent extends IHubResource, IItem {
   portalDataUrl?: string;
   /** The ids of any groups that the item belongs to */
   groupIds?: string[];
+  /**
+   * Any errors encountered when indexing or composing the content
+   * see https://github.com/ArcGIS/hub-indexer/blob/dc6146c3b7b4007b530f8e6357a9c36db59a6996/docs/errors.md#response-formatting-for-errors
```

Comment:
Maybe change to be the `master` link? `https://github.com/ArcGIS/hub-indexer/blob/master/docs/errors.md#response-formatting-for-errors`